c_examples/pause.c

```diff
@@ -1,17 +1,19 @@
+#define _DEFAULT_SOURCE
```

Comment:
What does this do?

Reply:
removes a warning os usleep definition:
https://stackoverflow.com/a/10053817/3017219